docs/customizing.md

```diff
@@ -8,7 +8,7 @@ following Dockerfile
 
 ```Dockerfile
 # The version of RancherOS to modify
-FROM rancher/os2:v0.0.1-test01
+FROM rancher-sandbox/os2:v0.0.1-test01
```

Comment:
shall we point maybe to a version that already exists? so copy-paste works ootb

Reply:
not sure, do we want to put an alpha in there? I would actually substitute the version for a $VERSION as usual and comment where to find all versions (repo/releases)